tools/build_updated_packages.rb

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env ruby
+# build_updated_packages version 1.0 (for Chromebrew)
+# This updates the versions in python pip packages by calling
+# tools/update_python_pip_packages.rb, checks for updated ruby packages
+# by calling tools/update_ruby_gem_packages.rb, and then checks if any
+# packages have been updated that need updated binaries. If so, it tries
+# to build those binaries.
+#
+# Author: Satadru Pramanik (satmandu) satadru at gmail dot com
+# Usage in root of cloned chromebrew repo:
+# tools/build_updated_packages.rb
+
+require_relative '../lib/color'
+require_relative '../lib/const'
+
+def require_gem(gem_name_and_require = nil, require_override = nil)
+  # Allow only loading gems when needed.
+  return if gem_name_and_require.nil?
+
+  gem_name = gem_name_and_require.split('/')[0]
+  begin
+    gem gem_name
+  rescue LoadError
+    puts " -> install #{gem_name} gem".orange
+    Gem.install(gem_name)
+    gem gem_name
+  end
+  requires = if require_override.nil?
+               gem_name_and_require.split('/')[1].nil? ? gem_name_and_require.split('/')[0] : gem_name_and_require
+             else
+               require_override
+             end
+  require requires
+end
+require_gem 'highline'
```

Comment:
Move to the top with the other require lines?

Reply:
Or will it not work until after the function is defined?

Reply:
The function is required before we use it...